.teamcity/Gradle_Check_dsl.iml

```diff
@@ -2,7 +2,7 @@
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
   <component name="FacetManager">
     <facet type="kotlin-language" name="Kotlin">
-      <configuration version="3" platform="JVM 1.6" useProjectSettings="false">
```

Comment:
Why is this changed? I ask because I don't know what this is for.

Reply:
Since IDEA changed it on my machine and I accidentally checked it in. On `master` we ignore the `.iml` file. Shall I cherry pick that commit as well?

Reply:
I cherry picked the ignoring as well.